lib/licensee/matchers/cran_matcher.rb

```diff
@@ -3,17 +3,31 @@ module Matchers
     class Cran < Package
       attr_reader :file
 
-      # While we could parse the package.json or bower.json file, prefer
+      # While we could parse the DESCRIPTION file, prefer
       # a lenient regex for speed and security. Moar parsing moar problems.
       LICENSE_REGEX = /
-        ^license:\s*([a-z\-0-9\.]+)
+        ^license:\s*(.+)
       /ix
 
       private
 
       def license_property
         match = @file.content.match LICENSE_REGEX
-        match[1].downcase if match && match[1]
+        return unless match && match[1]
+
+        # Remove The common + file LICENSE text
+        match = match[1]
+        match.slice!(/\s*\+\s*file\s*LICENSE.*/)
+
+        # Match GPL (>=2)
+        m = match.match(/^GPL\s*\(>=\s*([23])\)/)
+        return "gpl-#{m[1]}.0" if m && m[1]
```

Comment:
Actually no, the two options are [`GPL-2`](https://github.com/cran/ttutils/blob/bd83bbd6b0b7408502a1060da0274255343b9819/DESCRIPTION#L8), or [`GPL (>=2)`](https://github.com/cran/richards/blob/32f323299ab88c879f96d0906747adf98cdbca34/DESCRIPTION#L11)